services/dynamodb/src/main/java/software/amazon/awssdk/services/dynamodb/model/ItemBuilder.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2010-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.services.dynamodb.model;
+
+import static java.util.stream.Collectors.toList;
+
+import java.nio.ByteBuffer;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import software.amazon.awssdk.services.dynamodb.utils.AttributeValueHelper;
+import software.amazon.awssdk.utils.builder.SdkBuilder;
+
+public final class ItemBuilder implements SdkBuilder<ItemBuilder, Map<String, AttributeValue>> {
+
+    private Map<String, AttributeValue> item = new HashMap<>();
+
+    private ItemBuilder() {}
+
+    public static ItemBuilder builder() {
+        return new ItemBuilder();
+    }
+
+    @Override
+    public Map<String, AttributeValue> build() {
+        return item;
+    }
+
+    public ItemBuilder attr(String key, String stringValue) {
```

Comment:
Why do we want to shorten `attribute` here? 

Also, will this always have the same methods as `AttributeValueHelper` with a key? If so, is it better to just accept an `AttributeValue` and have them use the helper so that we don't duplicate it?